src/main/kotlin/com/expedia/graphql/schema/generator/WiringContext.kt

```diff
@@ -0,0 +1,9 @@
+package com.expedia.graphql.schema.generator
+
+/**
+ * Schema generation related information
+ */
+@Suppress("Detekt.EmptyClassBlock")
+class WiringContext internal constructor() {
```

Comment:
lets drop this for now as it is not needed

Reply:
The idea was that we can improve upon this later without a breaking API change.

Any extra information we'd pass to the developer in a later library update would require either changing the method or adding a new one.

Due to knowing graph-java's implementation and assuming they made smart decisions, we know it might be useful to provide more data for wiring purposes ([see here](https://github.com/ExpediaDotCom/graphql-kotlin/issues/60#issuecomment-438514404)).

The current use-case doesn't require it, so I'm okay with removing this if you want me to, I just thought that we'd want to avoid breaking API changes / cluttering the public interface.

Specifically the "NodeParentTree" might be interesting for #53, which I think can be implemented via custom directives.

What do you think?

Reply:
I've got two examples now that work without further information, so YAGNI?

A breaking change might be okay anyways seeing as this library is a `0.0.X` and a breaking change will make developers aware of new possibilities ;).

So I concur and will remove it.

Reply:
Can't actually find any examples using the extra information graphql-java provides, but maybe Google/Github search just sucks...

Reply:
Since we currently don't have a use case for use of those fields its hard to say whether we will need it or not. Let's drop it for now.

Reply:
80a629def32a9c130a5fc1b1778788217e471775